Zukini.UI/Steps/UISteps.cs

```diff
@@ -1,29 +1,37 @@
 ﻿using BoDi;
-using Coypu;
 using Zukini.Steps;
 
 namespace Zukini.UI.Steps
 {
-    public abstract class UISteps : BaseSteps
+    public abstract class UISteps : UISteps<StepsContext>
     {
+        public UISteps(IObjectContainer objectContainer) : base(objectContainer)
+        {
+            Context = new StepsContext(objectContainer);
```

Comment:
Will this work?
```
Context = objectContainer.Resolve<StepsContext>();
```

If new dependencies introduced in StepsContext - they will be autowired.

Reply:
good catch

Reply:
https://github.com/MikeMugu/Zukini/pull/36/commits/934a13a90530fbf1c9ffa3c7932b78bca1c3766b